components/home/home.component.tsx

```diff
@@ -0,0 +1,93 @@
+import * as React from 'react';
+import { withFormik, FormikProps, FormikErrors, Form, Field } from 'formik';
+import * as Yup from 'yup';
+
+interface FormValues {
+  email: string;
+  password: string;
+}
+
+interface OtherProps {
+  text?: string;
+}
+
+interface MyFormProps {
+  initialEmail?: string;
+  initialPassword?: string;
+}
+
+const InnerForm = (props: OtherProps & FormikProps<FormValues>) => {
+  const {
+    values,
+    errors,
+    touched,
+    handleChange,
+    handleBlur,
+    handleSubmit,
+    isSubmitting,
+    text,
+  } = props;
+
+  return (
+    <Form onSubmit={handleSubmit}>
+      <h3>{text}</h3>
+      <div>
+        <label>Email</label>
+        <Field
+          type='email'
+          name='email'
+          onChange={handleChange}
+          onBlur={handleBlur}
+          value={values.email}
+        />
+      </div>
+
+      <div>
+        <label>Password</label>
+        {touched.email && errors.email && <div>{errors.email}</div>}
+
+        <Field
+          type='password'
+          name='password'
+          onChange={handleChange}
+          onBlur={handleBlur}
+          value={values.password}
+        />
```

Comment:
Please create reusable input component using Baseweb for all input fields and buttons.

Reply:
okay